astro/src/content/docs/lifecycle/authenticate-users/single-sign-on.mdx

```diff
@@ -355,6 +355,17 @@ If you are looking to integrate with a commercial off the shelf or open source s
 
 Please see the [SAML IdP](/docs/lifecycle/authenticate-users/saml/) and [OIDC documentation](/docs/lifecycle/authenticate-users/oauth/) or the single sign-on documentation for the application you're looking to integrate with for more.
 
+### Bootstrapping SSO via Proxy
+<Aside type="version">Available in FusionAuth 1.56.0</Aside>
+
+In certain scenarios you may have authenticated a user outside of the typical OAuth flow. In such cases you can still acquire an SSO session from FusionAuth as long as you have a valid access token for the user. The way to do this is to redirect the browser to the `/oauth2/authorize` endpoint and provide the access token JWT as a Bearer token in the `Authorization` header. If the token is valid, FusionAuth will create an SSO session and redirect the browser back to the `redirect_uri` you provided.
+
+In order to use this feature you must enable the setting on the tenant either via the <InlineField>Allow access token bootstrap</InlineField> setting in the [Admin UI](/docs/get-started/core-concepts/tenants#oauth) or by setting the `tenant.ssoConfiguration.allowAccessTokenBootstrap` field to `true` in the [Tenant API](/docs/apis/tenants).
+
+Because browsers do not provide a way to set the `Authorization` header in a redirect, you must use a server-side proxy to make the request to FusionAuth. The proxy will need to set the `Authorization` header in the redirect of the browser to the `/oauth2/authorize` endpoint.
```

Comment:
```suggestion
Note that because browsers do not provide a way to set the `Authorization` header when browsing to a location, you will need to add that header using a reverse proxy or some other method.

The header will need to be in the following form:
<code-replaceme>
Authorization: Bearer <access token>
</code-replaceme>
```